source/content/guides/composer-convert.md

```diff
@@ -167,14 +185,44 @@ If you are using an exported config, you will need to move the configuration fil
 
 Locate the configuration files in your existing site and move them here. If they are stored in the files directory on your existing site, retrieve them via [SFTP](/sftp), as the Git clone would not contain them. The example project is configured to use this location.
 
+## Set up a Multidev (Optional)
+
+If your Pantheon account has access to [Multidev](/multidev), create a Multidev to push your new code to:
+
+```bash{promptUser:user}
+git push origin composerify && terminus env:create $site.dev composerify
+```
+
+This will set up the Multidev environment to receive and demo our changed code.
+
+## Create a New Composer Project
+
+1. In your local terminal, from the repository root of your Pantheon site, move a directory up:
+
+  ```bash{promptUser:user}
+  cd ..
+  ```
+
+2. Use Composer to create a new project, using the [Pantheon Drupal 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer) repository:
+
+    ```bash{promptUser:user}
+    composer create-project pantheon-systems/example-drops-8-composer $site-composer
+    cd $site-composer
+    ```
+
+This will create a new directory based on the example project [pantheon-systems/example-drops-8-composer](https://github.com/pantheon-systems/example-drops-8-composer) in the `$site-composer` directory.
+
+
 ## Update to Latest Drupal Core
+
 The Pantheon Drupal 8 Composer repository may not depend on the latest secure release of Drupal Core. To keep your site safe, it's a good idea to pull the latest version in now -- before moving on -- by running:
 
 ```bash{promptUser:user}
 composer update drupal/core --with-dependencies
 ```
 
 ## Prepare to Deploy
+
 At this point, your new project directory should contain all of the unique code from your existing Drupal 8 site, plus all of the code required to make a Composer driven project work. Since Pantheon requires all runtime code to be present when deploying to the platform, if no CI solution is a part of your workflow, you must now modify the project to be deployable straight to Pantheon.
```

Comment:
Heads up to update this part about if no CI solution is available and using Build Tools. I assume we want to direct folks to Integrated Composer as the default now.